dali/pipeline/operator/op_spec.h

```diff
@@ -441,16 +441,7 @@ class DLL_PUBLIC OpSpec {
                      "\" is not uniform. To access non-uniform argument inputs use "
                      "ArgumentWorkspace::ArgumentInput method directly.");
 
-    bool valid_shape =
-        shape.num_elements() == batch_size &&
-        (shape.num_samples() == batch_size || shape.num_samples() == 1);
-    if (should_throw) {
-      DALI_ENFORCE(valid_shape, make_string(
-                   "Unexpected shape of argument \"", name, "\". Expected batch of ", batch_size,
-                   " scalars or a batch of tensors containing one element per sample or a single "
-                   "tensor with ", batch_size, " elements. Got:\n", shape));
-    }
-    return valid_shape;
+    return true;
```

Comment:
This worries me. Why don't you do any verification? Where is this used?

Reply:
Fixed. Added removed lines back and added one more check to `operator.cc::EnforceUniformBatchSize`